### ISSUE TITLE
fix text in scores table

### ISSUE THREAD

**Issue by nickreich** [FluSightNetwork/cdc-flusight-ensemble#99]
The scores are now multi bin but the text still says single bin. Consider changing the text to be something like:

> The log-scores are computed by summing the predicted probabilities around a window of the eventually observed values. For the k-week-ahead targets, predictions within +/- 0.5 percentage points of the eventually observed value are considered accurate. For the targets on the scale of weeks, predictions within +/- 1 week of the eventually observed value are considered accurate. 

![image](https://user-images.githubusercontent.com/1280767/35491973-f8078240-0477-11e8-942f-9c7468485b00.png)

**Comment by lepisma** [comment]
Yes, this is in process. I am setting up a few pages documenting the scoring method and will be fixing this after linking that.

**Comment by lepisma** [comment]
Check now. I also added link to wiki's page on model evaluation.

**Comment by nickreich** [comment]
I don't see the link and there is an extra period at the end of one sentence. Otherwise, I think it looks good!

**Comment by lepisma** [comment]
Right, that extra period will go away in the next build. The link, right now, is in the score title
![image](https://user-images.githubusercontent.com/2487306/35537999-3d2c5932-0544-11e8-8b20-794960087194.png)

I believe another possible position to put the link in the main text is this?
![image](https://user-images.githubusercontent.com/2487306/35538038-68a0960a-0544-11e8-8b16-26d68c9e0d46.png)

**Comment by nickreich** [comment]
Sure, I like including both of those links.

-- 
Nicholas G. Reich, Ph.D.
Associate Professor
Department of Biostatistics and Epidemiology
School of Public Health and Health Sciences
University of Massachusetts, Amherst

413.545.4534
nick@schoolph.umass.edu
http://reichlab.io/
on twitter: @reichlab

**Comment by lepisma** [comment]
Done now.
